src/components/SMI-Table/SMITable.style.js

```diff
@@ -1,7 +1,8 @@
 import styled from "styled-components";
 
 export const TableWrapper = styled.div`
-overflow: hidden;
+overflow-x: auto;
```

Comment:
looks good to me. but the formatting of this style file is disturbed. it's not related to this change though